hoprd/rest-api/src/session.rs

```diff
@@ -446,10 +459,11 @@ pub(crate) async fn create_client(
             .await
             .contains_key(&ListenerId(protocol, bind_host))
     {
-        return Err((StatusCode::CONFLICT, ApiErrorStatus::InvalidInput));
+        return Err((StatusCode::CONFLICT, ApiErrorStatus::ListenHostAlreadyUsed));
```

Comment:
Also the body needs to be adapted in the swagger spec.

Reply:
Where ?

Reply:
Ok, you are correct in that we should stick to the generic approach.